src/content/devices/input-output/js/main.js

```diff
@@ -30,11 +30,11 @@ function gotDevices(deviceInfos) {
     option.value = deviceInfo.deviceId;
     if (deviceInfo.kind === 'audioinput') {
       option.text = deviceInfo.label ||
-        'microphone ' + (audioInputSelect.length + 1);
+      'microphone ' + (audioInputSelect.length + 1);
```

Comment:
From Sublime Text, at least, eslint seems to be a bit random about indenting :/.

Fixed.